stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -628,4 +638,76 @@ protected void closeInStream() {
       }
     }
   }
+
+  private final class AlluxioPOSIXBenchThread extends BenchThread {
+
+    private RandomAccessFile mRandomAccessFile = null;
+
+    private AlluxioPOSIXBenchThread(BenchContext context, int threadId) {
+      super(context, threadId);
+    }
+
+    @Override
+    @SuppressFBWarnings("BC_UNCONFIRMED_CAST")
+    protected int applyOperation() throws IOException, AlluxioException {
+      if (mRandomAccessFile == null) {
+        mRandomAccessFile = new RandomAccessFile(mFilePath.toString(), "rw");
+        mCurrentOffset = 0;
+      }
+      if (ClientIOOperation.isRead(mParameters.mOperation) && mParameters.mReadRandom) {
+        if (mParameters.mReadRandom) {
+          mCurrentOffset = mLongs.next();
+          mRandomAccessFile.seek(mCurrentOffset);
+        }
+      }
+      switch (mParameters.mOperation) {
+        case READ_ARRAY: // fall through
```

Comment:
RandomAccessFile doesn't have a function to pass in an offset, so it has to seek, making READ_ARRAY and READ_FULLY the same. Same for READ_FULLY and POS_READ_FULLY